modules/terraform-zscc-ccvm-gcp/main.tf

```diff
@@ -71,6 +71,21 @@ resource "google_compute_instance_group_manager" "cc_instance_group_manager" {
     replacement_method             = var.update_policy_replacement_method
   }
 
+  stateful_disk {
```

Comment:
The stateful MIG will not support ASG. This feature may need a toggle for the coming ASG.